extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -32,11 +32,31 @@ amp-sticky-ad {
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  overflow: visible !important;
+  direction: rtl;
```

Comment:
That will also work. Then user can stick it to left side by setting `left:0` later.